From: Didier Raboud <odyx@debian.org>
Date: Thu, 3 Dec 2020 12:32:54 +0100
Subject: SERVERBIN: match host_os_name on *-gnu first,
 to catch kfreebsd-gnu before *bsd*

Debian's kfreebsd-gnu architectures (kfreebsd-amd64 and kfreebsd-i386) work as Debian/GNU systems, hence with a /usr/lib/cups SERVERBIN, not a /usr/libexec/cups.

Bug-Upstream: https://github.com/OpenPrinting/cups/pull/60
---
 config-scripts/cups-directories.m4 | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/config-scripts/cups-directories.m4 b/config-scripts/cups-directories.m4
index b74083a..1430af3 100644
--- a/config-scripts/cups-directories.m4
+++ b/config-scripts/cups-directories.m4
@@ -262,6 +262,11 @@ AC_SUBST(CUPS_REQUESTS)
 
 # Server executables...
 case "$host_os_name" in
+	*-gnu)
+		# GNUs
+		INSTALL_SYSV="install-sysv"
+		CUPS_SERVERBIN="$exec_prefix/lib/cups"
+		;;
 	*bsd* | darwin*)
 		# *BSD and Darwin (macOS)
 		INSTALL_SYSV=""
